test/e2e/tainttoleration_test.go

```diff
@@ -92,14 +99,30 @@ var _ = ginkgo.Describe("propagation with taint and toleration testing", func()
 
 		ginkgo.AfterEach(func() {
 			ginkgo.By("removing taints in cluster", func() {
-				for _, cluster := range clusterNames {
-					clusterObj := &clusterv1alpha1.Cluster{}
-					err := controlPlaneClient.Get(context.TODO(), client.ObjectKey{Name: cluster}, clusterObj)
-					gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
-
-					clusterObj.Spec.Taints = nil
-					err = controlPlaneClient.Update(context.TODO(), clusterObj)
-					gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
+				for _, clusterName := range clusterNames {
+					taints := []corev1.Taint{
+						{
+							Key:    tolerationKey,
+							Value:  clusterName,
+							Effect: corev1.TaintEffectNoSchedule,
+						},
+					}
```

Comment:
Can we share the `trains with the one in `BeforeEach`?

Reply:
Here `clusterName` is a variable, I change to use 
```
func getReadyToAddTaints(tolerationKey, clusterName string) []corev1.Taint {
	return []corev1.Taint{
		{
			Key:    tolerationKey,
			Value:  clusterName,
			Effect: corev1.TaintEffectNoSchedule,
		},
	}
}
```
to get taints.

Reply:
rename func name to `constructAddedTaints`.